python/mxnet/_ctypes/symbol.py

```diff
@@ -115,7 +116,13 @@ def _set_symbol_class(cls):
     _symbol_cls = cls
 
 
-def _symbol_creator(handle, args, kwargs, keys, vals, name):
+def _set_np_symbol_class(cls):
```

Comment:
can we improve the docstring?

Reply:
In which way do you think the doc shall be improved? I think the current string is clearly describing what this function does.

Reply:
Added "numpy-compatible" to be more specific.